src/Mod/Part/App/Geometry.h

```diff
@@ -207,8 +218,11 @@ class PartExport GeomCurve : public Geometry
 
     Base::Vector3d value(double u) const;
 
+    GeomLine *toLine(bool clone=true) const;
+    GeomLineSegment *toLineSegment(bool clone=true) const;

```

Comment:
You'll no doubt be shocked to hear that I don't like leaving this as a boolean.

Reply:
Laugh!  Totally agree with this one.  The lack of meaning clarity is brutal.